src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue

```diff
@@ -0,0 +1,267 @@
+<template>
+	<www-page-minimal
+		:header-theme="headerTheme"
+		:footer-theme="footerTheme"
+	>
+		<div class="corporate-campaign-landing">
+			<section class="campaign-header section row align-center">
+				<div class="small-12 medium-10 large-6 xlarge-5 small-order-2 large-order-1 columns">
+					<no-click-loan-card />
+				</div>
+				<!-- eslint-disable-next-line max-len -->
+				<div class="small-10 large-6 xlarge-7 small-order-1 large-order-2 align-self-middle columns campaign-header__cta_wrapper">
+					<div v-if="headerLogo.url" class="campaign-header__logo">
+						<img :title="headerLogo.title" :src="headerLogo.url">
+					</div>
+					<hr v-if="headerLogo.url">
+					<h1 class="campaign-header__header">
+						{{ headline }}
+					</h1>
+					<div class="campaign-header__body" v-html="bodyCopy"></div>
+					<a
+						class="campaign-header__cta"
+						@click.prevent="jumpToLoans"
+						v-kv-track-event="[
+							'Campaign',
+							'click-hero-cta',
+							'Find someone to lend to',
+						]"
+					>
+						Find someone to lend to &xrarr;
+					</a>
+				</div>
+			</section>
+
+			<section class="campaign-status row align-center"></section>
+
+			<section class="campaign-loans row align-center"></section>
+
+			<section class="campaign-checkout row align-center"></section>
+		</div>
+	</www-page-minimal>
+</template>
+
+<script>
+import gql from 'graphql-tag';
+
+import { processPageContentFlat } from '@/util/contentfulUtils';
+import { applyUpcCode, applyRedemptionCode, applyLendingReward } from '@/util/campaignUtils';
+import { lightHeader, lightFooter } from '@/util/siteThemes';
+import WwwPageMinimal from '@/components/WwwFrame/WwwPageMinimal';
+import NoClickLoanCard from '@/components/Homepage/LendByCategory/NoClickLoanCard';
+import { documentToHtmlString } from '~/@contentful/rich-text-html-renderer';
+
+const pageQuery = gql`query pageContent($basketId: String!, $contentKey: String) {
+	contentful {
+		entries(contentType: "page", contentKey: $contentKey)
+	}
+	shop(basketId: $basketId) {
+		basket {
+			hasFreeCredits
+		}
+		lendingRewardOffered
+	}
+}`;
+
+export default {
+	inject: ['apollo'],
+	components: {
+		WwwPageMinimal,
+		NoClickLoanCard,
+	},
+	props: {
+		dynamicRoute: {
+			type: String,
+			default: ''
+		},
+		upc: {
+			type: String,
+			default: ''
+		},
+		promoCode: {
+			type: String,
+			default: ''
+		},
+		lendingReward: {
+			type: String,
+			default: ''
+		},
+	},
+	data() {
+		return {
+			headerTheme: lightHeader,
+			footerTheme: lightFooter,
+			pageTitle: 'Welcome',
```

Comment:
This is unused. You should be able to add the meta info block:
`	metaInfo() {
		return {
			title: this.pageTitle
		};
	},`

It would also be nice to get the title from contentful